LiveStream/Service/MockLiveStreamService.swift

```diff
@@ -94,13 +94,15 @@ internal struct MockLiveStreamService: LiveStreamServiceProtocol {
     }
   }
 
-  internal func subscribeTo(eventId: Int, uid: Int, isSubscribed: Bool)
-    -> SignalProducer<Bool, LiveApiError> {
+  internal func subscribeTo(eventId: Int, uid: Int, isSubscribed: Bool) -> SignalProducer<
+    LiveStreamSubscribeEnvelope, LiveApiError> {
```

Comment:
let's keep the type `SignalProducer<LiveStreamSubscribeEnvelope, LiveApiError>` all on one line